docs/GETTING_STARTED_ADD_PAT_OWNER.md

```diff
@@ -97,13 +57,6 @@ az devops configure -d project="$TEMPLATE_DEVOPS_PROJECT_NAME"
             | Cobalt-Contoso | If the aim is to use a single repository as ground truth for all potential patterns across your organization, effectively having to manage a combination of Cobalt patterns from a single repo, it's recommended to stick with a name that matches the project name. |
 
 
-> The following CLI command(s) can be run as an alternative to using the portal-based instructions:
-
-```bash
-az repos create --name "$TEMPLATE_DEVOPS_INFRA_REPO_NAME"
-az repos import create --git-url $COBALT_SOURCE_URL --repository "$TEMPLATE_DEVOPS_INFRA_REPO_NAME"
-```
-
 * Initialize new Azure Devops pipeline
     * Select Pipelines tab from within side-navigation menu
     * Select Create Pipeline and then choose 'Azure Repos Git [YAML]'
```

Comment:
Do you think the process of creating the repo and initializing a new pipeline need some screenshots as well similar to the error screenshot? Or would this be overkill?

Reply:
Screenshots for this step were included as a part of merging Neelam's (https://github.com/microsoft/cobalt/tree/negupt-bugFix) documentation branch.